toolchain/driver/codegen_options.cpp

```diff
@@ -23,6 +23,19 @@ documented there:
         arg_b.Default(host);
         arg_b.Set(&target);
       });
+
+  // TODO: We probably need a better infrstructure for forwarding flags to LLVM.
+  b.AddFlag(
+      {
+          .name = "pie",
+          .help = R"""(
+Toggles PIE output in LLVM.
+)""",
```

Comment:
Should we be exposing this? I think at least for now we could be opinionated here and say we'll always produce PIE binaries. (At some point we'll probably want a way to select PIC instead of PIE, but I think @chandlerc wanted us to think carefully about if and how we expose an ABI surface in a shared library and to only provide static linking in the common case, so PIE-only is probably good for now.)

Reply:
I'm agnostic. Done.